integration/vite-hmr-hdr-test.ts

```diff
@@ -112,6 +113,61 @@ test("Vite / HMR & HDR / mdx", async ({ page, viteDev }) => {
   expect(page.errors).toEqual([]);
 });
 
+test.describe("single fetch", () => {
```

Comment:
Didn't have copies of these tests with the flag enabled so didn't catch this until we migrated the changes to RR v7 where single fetch is always on